ovh/data_dbaas_logs_input_engine_test.go

```diff
@@ -1,33 +1,49 @@
 package ovh
 
 import (
+	"fmt"
+	"os"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
 const testAccDataSourceDbaasLogsInputEngine_deprecated = `
 data "ovh_dbaas_logs_input_engine" "logstash" {
- name          = "logstash"
- version       = "6.8"
- is_deprecated = true
+ service_name  = "%s"
+ name          = "%s"
+ version       = "%s"
+ is_deprecated = "%s"
 }
 `
+
 const testAccDataSourceDbaasLogsInputEngine_basic = `
 data "ovh_dbaas_logs_input_engine" "logstash" {
- name          = "logstash"
- version       = "7.x"
+ service_name  = "%s"
+ name          = "%s"
+ version       = "%s"
 }
 `
 
 func TestAccDataSourceDbaasLogsInputEngine_basic(t *testing.T) {
+	serviceName := os.Getenv("OVH_DBAAS_LOGS_SERVICE_TEST")
```

Comment:
Is it possible to add a precheck on this new environment variable `OVH_DBAAS_LOGS_SERVICE_TEST` please?
in order to skip the test and not fall in error if the env variable is not filled:

```
$ make testacc TESTARGS="-run TestAccDataSourceDbaasLogsInputEngine_deprecated"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run TestAccDataSourceDbaasLogsInputEngine_deprecated -timeout 240m
?   	github.com/ovh/terraform-provider-ovh	[no test files]
=== RUN   TestAccDataSourceDbaasLogsInputEngine_deprecated
    data_dbaas_logs_input_engine_test.go:77: Step 1/1 error: Error running pre-apply refresh: exit status 1

        Error: Error calling GET /dbaas/logs//input/engine/xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx:
        	 "HTTP Error 500: Server::InternalServerError: \"Internal server error\" (X-OVH-Query-Id: EU.ext-5.639b1a05.xxxx.xxxxx-xxxx-xxxx-xxxx-xxxxxxx)"

          with data.ovh_dbaas_logs_input_engine.logstash,
          on terraform_plugin_test.tf line 2, in data "ovh_dbaas_logs_input_engine" "logstash":
           2: data "ovh_dbaas_logs_input_engine" "logstash" {

--- FAIL: TestAccDataSourceDbaasLogsInputEngine_deprecated (3.94s)
FAIL
FAIL	github.com/ovh/terraform-provider-ovh/ovh	4.140s
?   	github.com/ovh/terraform-provider-ovh/ovh/helpers	[no test files]
testing: warning: no tests to run
PASS
ok  	github.com/ovh/terraform-provider-ovh/ovh/helpers/hashcode	(cached) [no tests to run]
FAIL
make: *** [testacc] Error 1
```

And add this ne env variable in the homepage please

Reply:
Should be better now